tasks/need_fixing_linters/golangci_lint_parser.py

```diff
@@ -0,0 +1,125 @@
+"""
+This module defines a parser for golangci-lint output.
+"""
+
+import re
+import os
+from codeowners import CodeOwners
+
+# example lint message
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ 
```suggestion
# Example lint message
```

Reply:
Thanks ! Fixed in [6d20653](https://github.com/DataDog/datadog-agent/pull/19015/commits/6d2065315f7ba06acbcea87bde309fd014ca4c61).